src/MaxText/__init__.py

```diff
@@ -26,10 +26,10 @@
 
 from collections.abc import Sequence
 
-from MaxText import maxtext_utils
-from MaxText import train_utils
-from MaxText import pyconfig
-from MaxText.layers import models
+from maxtext import maxtext_utils
+from maxtext import train_utils
+from maxtext import pyconfig
+from maxtext.layers import models
```

Comment:
@kanglant mentioned this folder should be `src/maxtext` now instead of `src/MaxText`. @SamuelMarks could you please check this? I guess you will still want the old one for the shim too